cmd/kubelet/app/server.go

```diff
@@ -154,8 +154,7 @@ via a flag.
 HTTP endpoint: HTTP endpoint passed as a parameter on the command line. This endpoint
 is checked every 20 seconds (also configurable with a flag).
 
-HTTP server: The kubelet can also listen for HTTP and respond to a simple API
-(underspec'd currently) to submit a new manifest.`,
+HTTP server: The kubelet can also listen for HTTP requests and respond to a simple API call to submit a new manifest.`,
```

Comment:
is it even true? I cannot find pointers. I see `NewSourceFile`, `NewSourceURL`, `NewSourceApiserver`, but not the server. I don't know the history here.

Can you please check if this is even true. If it is not, let's remove this statement everywhere

Reply:
Looks like `kubelet.md` was moved [content/en/docs/reference/command-line-tools-reference/kubelet.md](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/command-line-tools-reference/kubelet.md)
but, the description here does not seem to be consistent with `https://github.com/kubernetes/website/pull/299`

Reply:
It looks like this document was lost during the document migration.